### PR TITLE
Backend: Image Upload Functionality

### PR DESCRIPTION
**Pull request by Subham142**
Closes: #220 

#### Describe the changes you've made

A clear and concise description of what you have done to successfully close your assigned issue. Any new files? or anything you feel to let us know!

## Checklist:

- [x] My code follows the style guidelines of this project.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] I have made corresponding changes to the documentation.(Specially for Backend)
- [x] My changes generate no new warnings.

### CODE REVIEW

**Comment by Subham142** [comment]
@codewithvk  please review it
My largest contribution till now :)

File: client/src/Component/homepage/homepage.js [hunk 1]
```diff
@@ -0,0 +1,116 @@
+import React,{useState,useEffect} from "react";
```
Comment (codewithvk) [inline client/src/Component/homepage/homepage.js:1]: Why are you create a new Homepage component? You can refer [here](https://github.com/purnima143/Kurakoo/blob/master/client/src/components/homepage/homepage.js)

**Review by codewithvk — CHANGES_REQUESTED** [review]

**Comment by Subham142** [comment]
I really don't know why the homepage is showing here. I didn't do anything there
Should I paste the whole code from the updated repo into the homepage?

**Comment by codewithvk** [comment]
> ould I paste the

This happening because you don't fetch data.So first fetch data and push them again in PR :)

**Comment by Subham142** [comment]
Like this?

**Comment by codewithvk** [comment]
`npm pull `

**Comment by Subham142** [comment]
its says already up to date
Screenshot (106)

**Comment by Subham142** [comment]
> > I really don't know why the homepage is showing here. I didn't do anything there
> > Should I paste the whole code from the updated repo into the homepage?
> 
> This happening because you don't fetch data.So first fetch data and push them again in PR :)

done

File: client/src/Component/homepage/homepage.js (hunk 1 above)
Comment (codewithvk) [inline client/src/Component/homepage/homepage.js:1]: You can create a new component `client/src/components/homepage/homepage.js` this path, check `components` spelling.

**Review by codewithvk — CHANGES_REQUESTED** [review]

File: client/src/Component/homepage/homepage.js (hunk 1 above)
Comment (codewithvk) [reply client/src/Component/homepage/homepage.js:1]: or if you found already component of homepage then delete it :)

**Comment by Subham142** [comment]
@codewithvk  i deleted the wrong homepage
I reverte it and deleted the Componant/homepage

**Comment by codewithvk** [comment]
@Subham142 glad to merge your large contribution 🥇

**Comment by Subham142** [comment]
@codewithvk  i thought backend were lvl 3

**Comment by codewithvk** [comment]
@Subham142 Actually i am confused with level 2 and 3!

**Pull request merged**